packages/@aws-cdk/aws-redshift/lib/cluster.ts

```diff
@@ -454,6 +454,11 @@ export class Cluster extends ClusterBase {
 
     let loggingProperties;
     if (props.loggingBucket) {
+
+      if (!props.loggingKeyPrefix) {
+        throw new Error('Define logging key prefix used for logging bucket')
+      }
```

Comment:
The alternative to this approach would be to default the key prefix to `''`. This would (IMO) logically represent what the user wants by defaulting the property, and would result in a valid deployment rather than an error. What do you think?

Reply:
I initially thought about defaulting the property however, I believe that throwing an error before deployment is a better approach as it allows the developer to understand what properties need to be set for logging and allows them the option to set this property accordingly.

I have tested defaulting the key prefix to `''` as you have suggested and it does in fact result in a valid deployment.

Let me know what your thoughts are on this.

Reply:
Sorry, I missed this reply. 

> I believe that throwing an error before deployment is a better approach as it allows the developer to understand what properties need to be set for logging

I slightly disagree, because I don't think this property _should_ need to be set for logging. That being said, I don't feel strongly about it, and it's easier to change this error into defaulting to an empty string in the future than the reverse, so I'm fine keeping this feature as-is.

Can you please add a test for the new behavior?

Reply:
The main reason for this code update is to prevent the cluster from failing to create when the logging prefix is not set. If it not explicitly set, the cluster fails to create with "Internal Failure" which does not clarify what the issue is.

I have tested using an empty string and it does work as expected.